PayItForward.Data/Repositories/UsersRepository.cs

```diff
@@ -0,0 +1,129 @@
+﻿namespace PayItForward.Data
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.EntityFrameworkCore;
+    using PayItForward.Data.Models;
+    using Dbmodels = PayItForward.Data.Models;
+
+    public class UsersRepository<T, TKey> : IRepository<T, TKey>
+        where T : Dbmodels.User
+    {
+        public UsersRepository(PayItForwardDbContext context)
```

Comment:
USe Interface instead of class

Reply:
Instead of PayItForwardDbContext to use IPayItForwardDbContext ? Is that the point?